frontend/src/components/user/AccountForm.tsx

```diff
@@ -385,19 +456,22 @@ const AccountForm = ({
               {/* Volunteer fields */}
               <Box />
               {!isAdmin && (
-                <SelectorField
+                <SearchSelectorField
                   id="skills"
                   label="Skills"
                   values={values.skills}
                   options={skills}
-                  placeholder="Select Skills"
+                  addedValues={addedSkills}
+                  placeholder="Type to select skills or add new ones"
```

Comment:
Would it be more concise to say
```suggestion
                  placeholder="Add Skills"
```

Reply:
Good idea, done.